### ISSUE TITLE
Deployment to Azure with problems just on signin process

### ISSUE THREAD

**Issue by Ricardoaraujo00** [dotnet-presentations/blazor-workshop#223]
So i had created a free app service(F1) on Azure and made the deployment to that service. Everything works fine, but, when i click the signin button i get a 500 error.
I've the "Authentication:Twitter:ConsumerKey" and the "Authentication:Twitter:ConsumerSecret" in the app service configuration settings. I've tried with keys i got from a twitter development account but it didn't worked. Then i tried with the keys presented in the tutorial and it didn't worked(as said in the tutorial). I've also tried the development account keys in visual studio 2019 and it didn't worked, but worked with the tutorial keys.
In the app service i've also tried to add the keys in the Authentication/Autorixzation menu, and configured it to login with twitter. When i enforce login i get get a web page only with the sentence "The page cannot be displayed because an internal server error has occurred.", otherwise, i get the same 500 server error.
Does anyone knows what i'm missing out?
Thanks

**Comment by Ricardoaraujo00** [comment]
I just can't understand why the call back url doens't work on my twitter developer account. I've made a publish of the project and made it run on the localhost:64589 and the sign in process works fine with your key and secret. When i try to use the key and secret of my twitter account it doesn't works even if i specify the callback url as http://localhost:64589.

**Comment by Ricardoaraujo00** [comment]
Also this instructions dont work...
https://docs.microsoft.com/en-us/azure/app-service/configure-authentication-provider-twitter

**Comment by Ricardoaraujo00** [comment]
HOW, i really hate microsoft documentation! Besides it is hard to follow, seems that on top of that it is incorrect. I'll wait for some specialist to say something about this because i just wasted days of no productivity by not accomplishing this task and the last 6 hours trying to resolve this once for all. I'm very tired of working so hard on something so simple because of faulty documentation. This is horrible!!!!
So, the problem was, on twitter dev account, in the the call back address configuration, instead of putting "https://ricardoaraujo.azurewebsites.net/.auth/login/twitter/callback" as microsoft states in their documentation, it was as simple as "https://ricardoaraujo.azurewebsites.net/twitter-signin".
Shame on you microsoft.
And the hero that knows how to do this is Frank Boucher, and this is the video where i got to learn ho to do it: https://www.youtube.com/watch?v=WvtGgwa4lBY)
Microsoft should hire him!
And please, do at least correct documentation!!!!!!!!

**Comment by cylentwolf** [comment]
Microsoft did hire him.  If you know what is wrong make a pull request.

**Comment by FBoucher** [comment]
I'll put that in my todo and do a PR on MS docs as soon as I can. :)
